### PR TITLE
Fixes #50: Improve organization of “chart gallery”

### LINKED ISSUE #50: Improve organization of “chart gallery”
There is a [carousel for the 2023 30-day chart challenge](https://github.com/DOI-USGS/vizlab-home/blob/main/src/components/CC23_Carousel.vue), but other charts are organized sort of randomly. As this section grows, we’ll want to move them all to carousels so only a couple appear at once, but all can be browsed.  

- [x] Organize U.S. River Conditions
     - Released quarterly
     - Ideally the most recent one or two would appear and the rest could be found by navigating a carousel 
     - [PR](https://github.com/DOI-USGS/vizlab-home/pull/48) adding the most recent river conditions

- [x] Flow Tiles
     - Put in a carousel

Notes: 
 -Relates to / replaces #39 
-There's a [csv](https://github.com/DOI-USGS/vizlab-home/blob/main/public/viz-list.csv ) that controls everything on the landing page

Assignee: Maggie Jaenicke
(putting this here since I don't have repo access)

### PR DESCRIPTION
**Pull request by maggiejaenicke**
Changes made:
-----------

- Adds carousels for quarterly river conditions and monthly flow tiles. 

     - I decided to add river conditions videos directly into the site for a few reasons. The drupal link seems to error a lot, and I think maybe we consider replacing that slide click trigger (`handleSlickClick()` function on line 57 of `RiverConditions_Carousel.vue`) to drupal to go to twitter instead (or removing it altogether and just having the videos as they are in the site). When I had the carousel as just images, I could not get the thumbnails to stop opening in full screen without disabling full screen for other elements throughout the app which is why I pivoted to embedded videos. 

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [X] Chrome
- [ ] Safari
- [X] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/components/About.vue [hunk 1]
```diff
@@ -146,6 +146,16 @@ $coolBlue: rgb(66, 145, 235);
 
   }
 }
+.usa-nav__primary button[aria-expanded=false] span::after {  //removes those little squares that were appearing by Recent, Visualizations, About
```
Comment (cnell-usgs) [inline src/components/About.vue:149]: 🔥

File: src/components/About.vue [hunk 2]
```diff
@@ -146,6 +146,16 @@ $coolBlue: rgb(66, 145, 235);
...
+    background: url([object Object]) no-repeat center/contain;
+    display: inline-block;
+    height: 0px;
+    width: 0px;
+    content: "";
```
Comment (cnell-usgs) [inline src/components/About.vue:154]: Curious - what does this `content: ""` do?

File: src/components/PortfolioAccordions.vue [hunk 3]
```diff
@@ -72,6 +72,22 @@
           Chart Challenge 2023
         </h4>
         <ChartChallenge23Carousel />
+        <h4
+          id="head-river-conditions"
+        >
+          Quarterly River Conditions <a
```
Comment (cnell-usgs) [inline src/components/PortfolioAccordions.vue:78]:
```suggestion
          U.S. River Conditions (quarterly) <a
```

File: src/components/PortfolioAccordions.vue [hunk 4]
```diff
@@ -182,9 +202,10 @@
 
             // create groups
             this.vizList_interactives = this.vizList.filter((viz) => viz.group === this.vizGroups[0]); // all but river conditions and hurricanes
-            this.vizList_events = this.vizList.filter((viz) => viz.group === this.vizGroups[1]);
-            this.vizList_charts = this.vizList.filter((viz) => viz.group === this.vizGroups[2]); // static charts, twitter content
-
+            this.vizList_events_all = this.vizList.filter((viz) => viz.group === this.vizGroups[1]);
+            this.vizList_events = this.vizList_events_all.filter((viz) => !viz.title.includes("River Conditions"));
+            this.vizList_charts_all = this.vizList.filter((viz) => viz.group === this.vizGroups[2]); // static charts, twitter content
+            this.vizList_charts = this.vizList_charts_all.filter((viz) => !viz.title.includes("Flow cartogram")); //filters out streamflow tiles, can also just take them out of the viz_list eventually
```
Comment (cnell-usgs) [inline src/components/PortfolioAccordions.vue:208]: Add an issue to document this comment for future work

**Review by cnell-usgs — APPROVED** [review]
Nice job, this worked great for me: 
image

One thing I noticed that is cool, but may also slow the page down, is that each of the videos for river conditions can be played in the site. That's pretty nice. As there are more and more we will need to be smart about using lazy loading so that page doesn't load all of the videos at start. Similarly, the chart gallery charts use full size images for the thumbnails and load all charts when the carousel is clicked in. In a future improvement we should implement lazy loading to optimize the page. Please add an issue to note this, otherwise looks good to merge in.

File: src/components/About.vue (hunk 2 above)
Comment (maggiejaenicke) [reply src/components/About.vue]: Good catch! Looks like nothing. Removed.

**Comment by maggiejaenicke** [comment]
@cnell-usgs Made a few changes and opened issues based on your comments, so it should be good to merge in! I can't merge it myself because of my permissions on this repo.

**Pull request merged**